cves/CVE-2007-3847.yml

```diff
@@ -133,7 +142,9 @@ major_events:
 
     The event doesn't need to be directly related to this vulnerability, rather,
     we want to capture what the development team was dealing with at the time.
-  answer:
+  answer: |
```

Comment:
Was the code unchanged between the VCC and the fix, or were the changes insignificant/unrelated to the vulnerability?

Reply:
From what I saw in the git log, there were no significant changes to the code related to the vulnerability between the VCC and the fix.